### PR TITLE
problems: remove malus warning for qualifications

### PR DESCRIPTION
**Pull request by Elfikurr**
This should remove the malus warning for the qualification stage.

It doesn't remove the warning or force it to 0 though.

### CODE REVIEW

File: prologin/problems/models/submission.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 from prologin.models import CodingLanguageField
 from prologin.utils.db import MsgpackField
 from prologin.utils.rec_truncate import rec_truncate
+from contest.models import Event
```
Comment (zopieux) [inline prologin/problems/models/submission.py:19]: Let's keep this sorted maybe.

File: prologin/problems/models/submission.py [hunk 2]
```diff
@@ -178,12 +179,16 @@ def first_code_success(self):
         return self.codes.filter(score__gt=0).earliest()
 
     def score(self):
+        if (self.challenge_model().event_type == Event.Type.qualification):
```
Comment (zopieux) [inline prologin/problems/models/submission.py:182]: No useless parens please.

File: prologin/problems/views.py [hunk 3]
```diff
@@ -233,6 +233,7 @@ def get_context_data(self, **kwargs):
                                    .order_by('date_submitted')
                                    .last())
         context['user_submission'] = user_submission
+
```
Comment (zopieux) [inline prologin/problems/views.py:236]: Revert.

**Review by zopieux — CHANGES_REQUESTED** [review]

**Comment by Elfikurr** [comment]
@Zopieux thanks for the review, however, it was decided that this would not be added.
I forgot to close this PR, sorry.

**Pull request closed without merging**